php-fpm/Dockerfile

```diff
@@ -28,23 +27,42 @@ RUN apk add --update \
     php7-tokenizer \
     php7-xml \
     php7-zlib \
-    composer \
     libsodium \
     make \
-    curl
+    curl \
+    shadow \
+    tzdata \
+    sudo \
+    zsh \
+	git \
+	bash
 
+RUN cp /usr/share/zoneinfo/Europe/Paris /etc/localtime && echo "Europe/Paris" >  /etc/timezone && apk del tzdata
 RUN rm -rf /var/cache/apk/* && rm -rf /tmp/*
 
-ADD symfony.ini /etc/php7/php-fpm.d/
-ADD symfony.ini /etc/php7/cli/conf.d/
+RUN addgroup -g 1000 www-data && adduser -u 1000 -G www-data -D www-data -s zsh
+RUN groupmod -g 1000 www-data 
```

Comment:
Why create the `www-data` group a second time here ? It's already created line 43